2019-05-14 16:02:32,352 ERROR: Exception on /index [GET] [in c:\python27\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\python27\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\python27\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\python27\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\python27\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\python27\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "D:\microblog\app\routes.py", line 41, in index
    prev_url=prev_url)
  File "c:\python27\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "c:\python27\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "c:\python27\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\python27\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "D:\microblog\app\templates\base.html", line 48, in template
    {% block app_content %}{% endblock %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2019-05-14 16:02:49,411 ERROR: Exception on / [GET] [in c:\python27\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\python27\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\python27\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\python27\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\python27\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\python27\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\python27\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "D:\microblog\app\routes.py", line 41, in index
    prev_url=prev_url)
  File "c:\python27\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "c:\python27\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "c:\python27\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\python27\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "D:\microblog\app\templates\base.html", line 48, in template
    {% block app_content %}{% endblock %}
TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2019-05-14 16:03:41,575 INFO: Microblog startup [in D:\microblog\app\__init__.py:51]
